scripts/deploy.js

```diff
@@ -6,10 +6,12 @@
 
 /* eslint-disable no-console */
 const chalkTemplate = require('chalk');
-const { handler: build, builder: buildOptions } = require('./build');
+const { builder: buildOptions } = require('./build');
```

Comment:
The deploy script is still receiving the build options, but they are not used anymore